service/build.gradle

```diff
@@ -130,55 +144,50 @@ protobuf {
         // https://github.com/google/protobuf-gradle-plugin/issues/412
         // So we have to manually keep this version in sync, by referencing the BOM dashboard.
         // https://storage.googleapis.com/cloud-opensource-java-dashboard/com.google.cloud/libraries-bom/index.html
-        artifact = "com.google.protobuf:protoc:3.23.2"
+        artifact = 'com.google.protobuf:protoc:3.25.3'
     }
-    // By default, the proto plugin generates Java classes in the build/classes/ directory.
-    // generatedFilesBaseDir = "$projectDir/src/generated"
 }
 
-test {
-    useJUnitPlatform() {
-        excludeTags "regression-test"
-    }
-}
-
-task noCloudAccessRequiredTests(type: Test) {
+tasks.withType(Test).configureEach {
     useJUnitPlatform {
-        excludeTags "requires-cloud-access"
+        excludeTags 'regression-test'
     }
 }
 
-task regressionTests(type: Test) {
+tasks.register('regressionTests', Test) {
     useJUnitPlatform {
-        includeTags "regression-test"
+        includeTags 'regression-test'
     }
 }
```

Comment:
The regression test tasks doesn't seem to be running any tests. In the GHA run for this PR, I see:
```
The tag 'regression-test' is both included and excluded.  This will result in the tag being excluded, which may not be what was intended.  Please either include or exclude the tag but not both.
```

Reply:
(yikes). fixed